extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/YoutubeThrottlingParameterUtils.java

```diff
@@ -214,4 +237,17 @@ private static String validateFunction(@Nonnull final String function) {
         JavaScript.compileOrThrow(function);
         return function;
     }
+
+    @Nonnull
+    private static String fixupFunction(@Nonnull final String function)
```

Comment:
Mmmh, this function is specific to the new player code, right? Maybe it would be best to return the original `function` if the regexes don't find anything.

Also, could you add a comment explaining what `fixupFunction` does with an example?